Creiwry/irl-meeting-app
# fix(frontend): fix file naming issue with devise

### What does this PR do?
Fixes devise views generation error, adds parameter sanitizers to application controller for devise.

### Where should the reviewer start?
Check that views/devise folder exists, check Application controller and the presence of parameter sanitizers

### How should this be manually tested?
Check the views for devise. Go to /users/sign_in and /users/sign_up to check that the routes work.
